lib/rubocop/sequel/version.rb

```diff
@@ -4,7 +4,7 @@ module RuboCop
   module Sequel
     # This module holds the RuboCop Sequel version information.
     module Version
-      STRING = '0.3.5'
+      STRING = '0.3.6'
```

Comment:
Let me know if you'd rather I not manually update the version numbers in this PR

Reply:
I think that's fine considering we'll ship this shortly after.